apps/test-app/package.json

```diff
@@ -15,10 +16,10 @@
     "@graphql-codegen/typescript": "^2.4.8",
     "@graphql-codegen/typescript-resolvers": "^2.6.1",
     "@types/node": "^14",
-    "aws-cdk-lib": "2.43.1",
     "aws-cdk": "2.43.1",
+    "aws-cdk-lib": "2.43.1",
     "constructs": "10.0.0",
-    "functionless": "file:../",
+    "functionless": "*",
```

Comment:
`yarn install` in the root of the project will automatically treat this as a linked dependency.

Reply:
How will this work for a non-private package? The version range could matter and there may be peer dependencies

Reply:
This is outside the scope of turborepo and depends on the versioning/publishing solution we choose.

See [versioning and publishing][1] for more info.

[1]: https://turborepo.org/docs/handbook/publishing-packages/versioning-and-publishing